File: apps/muly-node-js-admin/src/api/book/BookCreateInput.ts
import { AuthorWhereUniqueInput } from "../author/AuthorWhereUniqueInput";

export type BookCreateInput = {
  title?: string | null;
  description?: string | null;
  isbn?: string | null;
  publishedDate?: Date | null;
  author?: AuthorWhereUniqueInput | null;
};

File: apps/muly-dotnet-server/src/APIs/Book/Dtos/BookCreateInput.cs
namespace MulyDotnet.APIs.Dtos;

public class BookCreateInput
{
    public string? Id { get; set; }

    public DateTime CreatedAt { get; set; }

    public DateTime UpdatedAt { get; set; }

    public string? Title { get; set; }

    public string? Description { get; set; }

    public string? Isbn { get; set; }

    public DateTime? PublishedDate { get; set; }

    public AuthorIdDto? Author { get; set; }
}
